### PR TITLE
Add attachment info to pg:credentials

### PR DESCRIPTION
**Pull request by camillebaldock**
## What is this?

* Provide attachment information in the list of credentials
* Order default first

## Notes

This PR is based on the `pg-credentials` branch, once the `pg-credentials` branch is merged, I will point it to master.

### CODE REVIEW

**Review by genslein — APPROVED** [review]

File: commands/credentials.js [hunk 1]
```diff
@@ -11,12 +13,57 @@ function * run (context, heroku) {
...
+      )
+
+      // render each attachment under the credential
+      let attLines = atts.map(function (attachment, idx) {
+        let isFirst = (idx === credAttachments.length - 1)
+        return renderAttachment(attachment, app, isFirst)
+      })
+
+      return [cred].concat(attLines).join('\n')
+    }
```
Comment (msakrejda) [inline commands/credentials.js]: This seems pretty fiddly. The code used by `addons` to do basically the same thing is not reusable here?

File: commands/credentials.js [hunk 2]
```diff
@@ -11,12 +13,57 @@ function * run (context, heroku) {
...
+      let isForeignApp = (attOrAddon) => attOrAddon.app.name !== app
+      let atts = sortBy(credAttachments,
+        isForeignApp,
+        'name',
+        'app.name'
+      )
+
+      // render each attachment under the credential
+      let attLines = atts.map(function (attachment, idx) {
+        let isFirst = (idx === credAttachments.length - 1)
```
Comment (msakrejda) [inline commands/credentials.js]: Technically `isLast`, no?

**Review by msakrejda — APPROVED** [review]
+1

File: commands/credentials.js (hunk 1 above)
Comment (camillebaldock) [reply commands/credentials.js]: renderAttachment is straight copy from there, unfortunately the ordering of credentials if differerent that in addons, and has the special case of `default` so it wasn't quite 100% reusable; where would be put shared-ish code between those CLI plugins?

File: commands/credentials.js (hunk 2 above)
Comment (camillebaldock) [reply commands/credentials.js]: yes!, funnily enough that wrong name is straight from heroku-cli-addons: fixing it here

**Pull request merged**